website/docs/docs/cloud/cloud-cli-installation.md

```diff
@@ -1,107 +1,161 @@
+
+After installing, you invoke the Cloud CLI the same way you would call dbt Core.  For example, to compile a project using dbt Cloud:
+
+```bash
+dbt deps
+dbt compile
+```
+
+
+## Installing the dbt Cloud CLI
+
+:::caution
+
+For compatibility, both the Cloud CLI and dbt Core are invoked by running `dbt`. This can cause a path conflict, where your OS invokes either the Cloud CLI or dbt Core depending on it's presence in your $PATH environment variable.
 
-### Install and update with Brew on MacOS (recommended)
+If you have dbt Core installed locally, ensure that you either deactivate your Python environment or uninstall before proceeding.  Alternatively, advanced users can modify the $PATH environment variable to correctly point to the dbt Cloud CLI binary to use both the Cloud CLI and dbt Core together.
 
-1. Install the dbt Cloud CLI: 
+You can always uninstall the Cloud CLI to return to using dbt Core.
+:::
+
+### Install and update with Brew - MacOS
+
+1. Verify that there is no conflict with a dbt Core installation on your system. 
+
+```bash
+which dbt
+# Should return "dbt not found"
+```
+
+If the dbt help text appears, deactivate your current dbt Core install.
+
+2. Install the dbt Cloud CLI with Brew: 
 
 ```bash
 brew tap dbt-labs/dbt-cli
 brew install dbt-cloud-cli
 ```
 
-2. Verify the installation by requesting your homebrew installation path (not your dbt core installs). If the `which dbt` command returns nothing, then you should modify your PATH in `~.zshrc` or create an alias.
+3. Verify the installation. If the help text does not indicate that you're using the dbt Cloud CLI, make sure you've deactivated your pyenv or venv and do not have a version of dbt globally installed.
 
 ```bash
-which dbt
 dbt --help
 ```
 
-### Manually install (Windows and Linux)
+#### Updating your dbt Cloud installation - MacOS
 
-1. Download the latest release for your platform from [GitHub](https://github.com/dbt-labs/dbt-cli/releases).
-2. Add the `dbt` executable to your path.
-3. Move to a directory with a dbt project, and create a `dbt_cloud.yml` file containing your `project-id` from dbt Cloud.
-4. Invoke `dbt --help` from your terminal to see a list of supported commands.
+Run `brew upgrade dbt-cloud-cli` to update the Cloud CLI.  During the beta period, we recommend updating before filing a bug report, as the API is subject to breaking changes!
 
-#### Updating your dbt Cloud installation (Windows + Linux)
 
-Follow the same process in [Installing dbt Cloud CLI](#manually-install-windows-only) and replace the existing `dbt` executable with the new one. You should not have to go through the security steps again.
+### Manually install - Windows
 
-## Setting up the CLI
+1. Download the latest Windows release for your platform from [GitHub](https://github.com/dbt-labs/dbt-cli/releases).
 
-The following instructions are for setting up the dbt Cloud CLI. 
+2. Extract the `dbt-cloud-cli.exe` file into the same folder as your dbt project.
```

Comment:
since there are many options, how do users know what release asset to download? for instance, if they have windows, do they download dbt_0.29.9_windows_amd64.tar.gz, or dbt_0.29.9_windows_arm64.tar.gz, or dbt_0.29.9_windows_armv6.tar.gz? when i downloaded those three, i didn't see 'dbt-cloud-cli.exe'. i saw 'dbt.exe'.